Common/include/option_structure.hpp

```diff
@@ -762,6 +762,7 @@ enum BC_TYPE {
   LOAD_DIR_BOUNDARY = 35,		/*!< \brief Boundary Load definition. */
   LOAD_SINE_BOUNDARY = 36,		/*!< \brief Sine-waveBoundary Load definition. */
   NRBC_BOUNDARY= 37,   /*!< \brief NRBC Boundary definition. */
+  FLUID_INTERFACE = 38,	/*!< \brief Domain interface definition. */
```

Comment:
I think we'll have a conflict here with Francisco's internal marker set to 38 in his current pull request. Can you please set this to 39?